### ISSUE TITLE
Accessibility: "Save Settings" Button Contrast Fails WebAIM Check

### ISSUE THREAD

**Issue by bobmatyas** [Automattic/jetpack#21734]
### Impacted plugin

Jetpack

### Steps to Reproduce

In the Jetpack settings pages, the "Save Settings" buttons don't pass [the WebAIM contrast test](https://webaim.org/resources/contrastchecker/):

`https://wordpresssite.com/wp-admin/admin.php?page=jetpack#/writing`

![Annotation on 2021-11-12 at 15-58-52](https://user-images.githubusercontent.com/45246438/141533725-8407a26c-35c2-476e-a13b-ea60f16e1578.png)

### A clear and concise description of what you expected to happen.

Color contrast should pass WebAIM checks.

### What actually happened

The color combo `#66C6E4` on `#008EC2` fails the contrast test.

### Other information

<img width="746" alt="Screenshot on 2021-11-12 at 15-51-53" src="https://user-images.githubusercontent.com/45246438/141533296-4f3aad05-9a72-4d46-801d-a1ffacdc43bf.png">

### Operating System

macOS

### Browser

Firefox

**Comment by jeherve** [comment]
Closing as duplicate of #16452
